tests/Unit/ProtoTests/ProtoTest.php

```diff
@@ -150,4 +150,8 @@ public function testDisableSnippetGeneration(): void
             false
         );
     }
+
+    public function testBasicAutoPopulation(): void {
```

Comment:
PHP code standards - no cuddly braces for class methods
```suggestion
    public function testBasicAutoPopulation(): void
    {
```